docs/api/IoT.js-API-ADC.md

```diff
@@ -10,47 +10,32 @@ The following shows ADC module APIs available for each platform.
 | adcpin.close | O | O | O |
 | adcpin.closeSync | O | O | O |
 
-
-## Contents
-* [Pin](#pin)
-* [ADC](#adc)
-  * [Constructor](#adc-constructor)
-    * [`new ADC()`](#adc-new)
-  * [Prototype methods](#adc-prototype-methods)
-    * [`adc.open(configuration[, callback])`](#adc-open)
-* [ADCPin](#adcpin)
-  * [Prototype methods](#adcpin-prototype-methods)
-    * [`adcpin.read([callback])`](#adcpin-read)
-    * [`adcpin.readSync()`](#adcpin-read-sync)
-    * [`adcpin.close([callback])`](#adcpin-close)
-    * [`adcpin.closeSync()`](#adcpin-close-sync)
-
-## `pin` <a name="pin"></a>
-On Nuttx, you have to know pin number that is defined in target board module. For more module information, please see below list.
+### `pin`
+On Nuttx, you have to know the number of pins that is defined on the target board module. For more information, please see the list below.
   * [STM32F4-discovery](../targets/nuttx/stm32f4dis/IoT.js-API-Stm32f4dis.md#adc-pin)
 
-## Class: ADC <a name="adc"></a>
+# Class: ADC
 
 
-## Constructor <a name="adc-constructor"></a>
+## Constructor
 
 
-### `new ADC()` <a name="adc-new"></a>
+### `new ADC()`
 
-Returns a new ADC object which can open ADC pin.
+Returns a new ADC object which can open an ADC pin.
 
 
-## Prototype methods <a name="adc-prototype-methods"></a>
+## Prototype methods
 
 
-### `adc.open(configuration[, callback])` <a name="adc-open"></a>
-* `configuration <Object>`
-  * `device <String>`, mandatory configuration on Linux
-  * `pin <Number>`, mandatory configuration on Nuttx
-* `callback <Function(err: Error | null)>`
-* Returns: `<ADCPin>`
+### adc.open(configuration[, callback])
+* `configuration` {Object}
+  * `device` {String} mandatory configuration on Linux
+  * `pin` {Int} mandatory configuration on Nuttx
+* `callback` {Function(err: Error | null)}
```

Comment:
The correct style is:
* `callback` {Function}
  * `err` {Error|null} 
Please fix all instances.